ntbs-service/Pages/Shared/_AlertsAndActions.cshtml

```diff
@@ -10,31 +10,40 @@
     @if ((string)ViewData["CurrentPage"] == NotificationSubPaths.Overview)
     {
         <nhs-grid-column grid-column-width="TwoThirds">
-            <div class="@manageNotificationClass">
-                <h2>
-                    Alerts
-                </h2>
-                @if(Model.Alerts.Count > 0)
-                {
-                    <div class="overview-alerts-container">
-                    @foreach (var alert in Model.Alerts)
+            @if(!Model.HasEditPermission)
+            {  
+                <!-- provide non breaking space so that nhs-grid-column is rendered and pushes manage notification
+                    to the right when alerts are hidden -->
+                @:&nbsp;
```

Comment:
Alternative approach would be applying a justify-content: flex-end to the manage notification section grid column, if anyone has a preference I'm happy to be persuaded. Currently nhs-grid-columns aren't rendered if they don't have any content

Reply:
Exactly right - if you leave it in then the spacing when alerts are shown doesn't look great